settings.gradle.kts

```diff
@@ -27,3 +27,5 @@ include("jacodb-benchmarks")
 include("jacodb-cli")
 include("jacodb-approximations")
 include("jacodb-taint-configuration")
+include("jacodb-ifds:actors")
+include("jacodb-ifds:ifds")
```

Comment:
If you are proposing to start using nested modules (which might be a pain to use in some cases), why not put "actors" and "ifds" as submodules of "jacodb-analysis"? That is, consider making a single module "jacodb-analysis" (it is already here), move everything from it to "jacodb-analysis:taint" (as it is taint-specific), and add your two modules to it. Or better just put everything into one gradle module "jacodb-analysis" - what is the value of splitting the codebase into nested submodules here?